util-logging/src/main/scala/com/twitter/logging/FileHandler.scala

```diff
@@ -166,8 +166,10 @@ class FileHandler(
   }
 
   private def openStream(): OutputStream = {
-    val dir = new File(filename).getParentFile
+    val file = new File(filename)
+    val dir = file.getParentFile
     if ((dir ne null) && !dir.exists) dir.mkdirs
+    bytesWrittenToFile = if (file.exists()) file.length() else 0
```

Comment:
Yep, I understand both of those points and I still think it's awkward. I'll drop this if nobody else has concerns though.